packages/wast-parser/test/tokenizer/number-literal-errors/multiple-signs/throws.txt

```diff
@@ -0,0 +1,9 @@
+(module
+  (func
+    (i32.const ++-----1)
+               ^
+  )
+)
+
+
+Unexpected character "+"
```

Comment:
:heart_eyes: lgtm

Reply:
Wow thats really cool! :tada: Is the `^` off by one because of location issues with line/col?

Reply:
I think it's because we throw before incrementing the col or something because some are correct.